pkgs/desktops/plasma-5/kwin/default.nix

```diff
@@ -21,20 +23,27 @@ mkDerivation {
   pname = "kwin";
   nativeBuildInputs = [ extra-cmake-modules kdoctools ];
   buildInputs = [
-    libepoxy lcms2 libICE libSM libcap libdrm libinput libxkbcommon mesa pipewire
-    udev wayland xcb-util-cursor xwayland
+    libepoxy lcms2 libICE libSM libcap libdrm libinput libxkbcommon libxcvt
+    mesa pipewire plasma-wayland-protocols udev wayland wayland-protocols
+    xcb-util-cursor xwayland
```

Comment:
same here

Reply:
moving all additions to a separate line + keep current formatting